.meteor/versions

```diff
@@ -137,6 +137,7 @@ rocketchat:crowd@1.0.0
 rocketchat:custom-oauth@1.0.0
 rocketchat:custom-sounds@1.0.0
 rocketchat:dolphin@0.0.2
+rocketchat:drupal@0.0.1
```

Comment:
Can you undo all other changes to this file but this line here?  Ideally this PR shouldn't be updating other packages, it should only be adding the code needed for drupal.  We'd handle the updating of packages seperately